plugins/linux-v4l2/v4l2-input.c

```diff
@@ -1025,6 +1027,11 @@ static void v4l2_init(struct v4l2_data *data)
 	blog(LOG_INFO, "Linesize: %d Bytes", data->linesize);
 
 	/* set framerate */
+	if (data->framerate == 0) {
```

Comment:
I think it got mixed when you were applying my suggestions. We should move this after the v4l2_set_framerate (tries to apply the chosen fps and returns the resulting fps, which may be 0) and before the unpack on L1039.

Reply:
whoops. should be fixed now